### DISCUSSION TITLE
Put password to root and now Permission denied

### DISCUSSION THREAD

**Discussion by cybercuate** [AzuraCast/AzuraCast#5898] [Q&A]
Hi, I use azuracast and RDP on lxde on oracle ARM64 vps. It was working good until I decided to login via RDP as root, so I had to put a password to the root user. I did nothing else and now I get the error on azuracast "permission denied" when I try to upload music (via SFTP or web browser) or even creating a new directory. Can you help me please?. I can no longer upload songs.

**Comment by BusterNeece** [comment]
@cybercuate This was a bug we had in a Rolling Release version a few days/weeks ago...perhaps updating could help resolve this?

**Reply by cybercuate** [reply]
> @cybercuate This was a bug we had in a Rolling Release version a few days/weeks ago...perhaps updating could help resolve this?

I just updated from 0.17.4 to 0.17.5 and the problem is still there. I tried rebooting the server and no results. If it is a bug, I can report it wasn't fixed as of 0.17.5 stable release
